test/e2e/escrow.fingerprint.e2e-spec.ts

```diff
@@ -137,6 +137,18 @@ describe('EscrowController (e2e) using fingerprint plugin', () => {
             });
     });
 
+    it('Recreate same endpoint', () => {
+        body.id = 'agentIdxyz';
```

Comment:
Shouldn't need to set this again since the id is already set after the previous test.